recipes/yacman/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "yacman" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "37cd49463880f4c9cfee0dc8280bc893fc3bf392615d12b35c97609def71071c"
```

Could you please remove the quotes from ``sha256``?
```suggestion
  sha256: 37cd49463880f4c9cfee0dc8280bc893fc3bf392615d12b35c97609def71071c
```